lms/models/lis_result_sourcedid.py

```diff
@@ -0,0 +1,72 @@
+import datetime
+
+import sqlalchemy as sa
+
+from lms.db import BASE
+
+
+__all__ = ["LISResultSourcedId"]
+
+
+class LISResultSourcedId(BASE):
+    """
+    A record of a student's launch of a Hypothesis-configured LMS assignment
```

Comment:
Long docstring here, but I find this whole concept pretty murky/opaque and thought future devs might as well.